please/please.py

```diff
@@ -11,172 +11,192 @@
-def add(task: str):
-    new_task = {
-        "name": task,
-        "done": False
-    }
+@app.command(short_help="Add a Task")
+def add(task: str) -> None:
+    new_task = {"name": task, "done": False}
     config["tasks"].append(new_task)
     write_config(config)
-    center_print(f"Added \"{task}\" to the list",
-                 "cyan1 on purple3")
+    center_print(f'Added "{task}" to the list', "cyan1 on purple3")
     print_tasks(config["tasks"])
 
 
-@ app.command(short_help='Deletes a Task')
-def delete(index: int):
+@app.command(short_help="Deletes a Task")
+def delete(index: int) -> None:
     index = index - 1
-    if index >= len(config["tasks"]):
+    if len(config["tasks"]) == 0:
+        center_print_wrap("Task list is empty", style="bright_red on bright_white")
+    elif not 0 <= index < len(config["tasks"]):
         center_print_wrap(
-            "Are you sure you gave me the correct number to delete?", "bright_red on bright_white")
-        return
-
-    if len(config["tasks"]) > 0:
+            "Are you sure you gave me the correct number to delete?",
+            "bright_red on bright_white",
+        )
+    else:
         deleted_task = config["tasks"][index]
         del config["tasks"][index]
         write_config(config)
         center_print(f"Deleted '{deleted_task['name']}'", "cyan1 on purple3")
         print_tasks(config["tasks"], True)
-    else:
-        center_print_wrap(
-            "Sorry, I've got no tasks to delete", style="bright_red on bright_white")
 
 
-@ app.command(short_help='Mark a task as done')
-def done(index: int):
+@app.command(short_help="Mark a task as done")
+def done(index: int) -> None:
     index = index - 1
-    if index >= len(config["tasks"]):
-        center_print_wrap(
-            "Are you sure you gave me the correct number to mark as done?", "bright_red on bright_white")
-        return
 
-    if len(config["tasks"]) > 0 and not all_tasks_done():
+    if len(config["tasks"]) == 0:
+        center_print_wrap(
+            "Task list is empty",
+            style="bright_red on bright_white",
+        )
+    elif all_tasks_done():
+        center_print("All tasks are already completed!", "black on green")
+    elif not 0 <= index < len(config["tasks"]):
+        center_print_wrap(
+            "Are you sure you gave me the correct number to mark as done?",
+            "bright_red on bright_white",
+        )
+    else:
         config["tasks"][index]["done"] = True
         write_config(config)
         center_print("Updated Task List", "black on green")
         print_tasks(config["tasks"])
-    else:
-        center_print_wrap(
-            "Sorry, I've got no tasks to mark as done", style="bright_red on bright_white")
 
 
-@app.command(short_help='Mark a task as undone')
-def undone(index: int):
+@app.command(short_help="Mark a task as undone")
+def undone(index: int) -> None:
     index = index - 1
-    if index >= len(config["tasks"]):
+    if len(config["tasks"]) == 0:
```

Comment:
The conditions aren't mutually exclusive, it's better not to use if/elif here (mutually exclusive conditions should be handled in their own if/elif statements), so would have to be reverted unfortunately.

Reply:
make sense. will change wherever applicable.